### PR TITLE
Refactor tools Dockerfiles, docker_run.sh scripts

### PR DESCRIPTION
**Pull request by zkokelj**
### Why this change is needed

Standardise Dockerfiles and `docker_run.sh` commands for all the tools.

(The only exception is `hardhatdeployer`, because it doesn't need `docker_run,sh`)

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/obscuroscan/container_run.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            --logPath)           logPath=${value} ;;
+            --port)              port=${value} ;;
+            --image)             image=${value} ;;
+            *)
+    esac
+done
+
+# Stop and remove any running container, and then star
+echo "Force stopping any existing container ... "
+docker rm -f  local-testnet-obscuroscan 2>/dev/null
```
[the remark this answers is not in the document]
Comment (zkokelj) [reply tools/obscuroscan/container_run.sh:37]: I agree, renamed 👍

File: tools/obscuroscan/container_run.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# This script starts Obscuroscan for testnet
+#
+
+# Ensure any fail is loud and explicit
+set -euo pipefail
+
+# Define defaults
+nodeID=""
+rpcServerAddress="http://testnet.obscu.ro:13000"
+address="127.0.0.1:3000"
```
[the remark this answers is not in the document]
Comment (zkokelj) [reply tools/obscuroscan/container_run.sh:13]: This `address` is the default value for `Obscuroscan` on which the server will run.
(https://github.com/obscuronet/go-obscuro/blob/main/tools/obscuroscan/main/cli.go#L16, 
https://github.com/obscuronet/go-obscuro/blob/main/tools/obscuroscan/main/main.go#L19)

File: .github/workflows/manual-deploy-testnet-l2.yml [hunk 3]
```diff
@@ -98,8 +98,6 @@ jobs:
           docker push ${{env.L2_ENCLAVE_DOCKER_BUILD_TAG}}
           DOCKER_BUILDKIT=1 docker build -t ${{env.L2_HOST_DOCKER_BUILD_TAG}} -f dockerfiles/host.Dockerfile .
           docker push ${{env.L2_HOST_DOCKER_BUILD_TAG}}
-          DOCKER_BUILDKIT=1 docker build -t ${{env.L2_CONTRACTDEPLOYER_DOCKER_BUILD_TAG}} -f testnet/contractdeployer.Dockerfile .
-          docker push ${{env.L2_CONTRACTDEPLOYER_DOCKER_BUILD_TAG}}
```
Comment (otherview) [inline .github/workflows/manual-deploy-testnet-l2.yml:102]: Where did the contract deployer moved to ?

File: testnet/docker-compose.local.yml [hunk 4]
```diff
@@ -12,11 +12,6 @@ services:
     build:
       context: $ROOT_PATH
       dockerfile: ./dockerfiles/host.Dockerfile
-  contractdeployer:
```
Comment (otherview) [inline testnet/docker-compose.local.yml:15]: This image is not used anywhere else ?

**Review by StefanIliev545 — APPROVED** [review]

**Pull request merged**